scripts/generate-change-list.js

```diff
@@ -149,6 +149,6 @@ if (require.main === module) {
     });
 }
 
-module.exports = {
-  changelistSinceTag,
-};
+if (import.meta.url.endsWith(process.argv[1])) {
+  main();
+}
```

Comment:
Taken from https://stackoverflow.com/a/75389579/434243. This could produce false positives in principle, but it is simple and I think we can get away with it in practice.